pkg/kube/kubehttp.go

```diff
@@ -21,18 +21,25 @@ import (
 	apiruntime "k8s.io/apimachinery/pkg/runtime"
 )
 
-func CreateScheme() *apiruntime.Scheme {
+func CreateScheme() (*apiruntime.Scheme, error) {
 	scheme := apiruntime.NewScheme()
-	_ = sourcev1.AddToScheme(scheme)
-	_ = kustomizev2.AddToScheme(scheme)
-	_ = helmv2.AddToScheme(scheme)
-	_ = corev1.AddToScheme(scheme)
-	_ = extensionsv1.AddToScheme(scheme)
-	_ = appsv1.AddToScheme(scheme)
-	_ = rbacv1.AddToScheme(scheme)
-	_ = authv1.AddToScheme(scheme)
-
-	return scheme
+	builder := apiruntime.SchemeBuilder{
+		sourcev1.AddToScheme,
+		kustomizev2.AddToScheme,
+		helmv2.AddToScheme,
+		corev1.AddToScheme,
+		extensionsv1.AddToScheme,
+		appsv1.AddToScheme,
+		rbacv1.AddToScheme,
+    authv1.AddToScheme,
```

Comment:
Maybe gofmt this file?

Reply:
GitHub messed it up when I merged in changes from main and had to resolve the merge conflict.